clients/vscode-hlasmplugin/README.md

```diff
@@ -179,7 +175,7 @@ The program name in `pgm_conf.json` can be wildcarded, as in the following examp
 ```
 In this example, GROUP1 is used for all open code programs.
 
-Assmebler options defined by the processor group can be overriden in the `pgm_conf.json` file as shown in the following example:
+You can override Assembler options, that are defined by the processor group, in the `pgm_conf.json` file as shown in the following example:
```

Comment:
We tend to address the user directly for better readability rather than using the passive voice

Reply:
From the original wording, one can infer that Assembler options can be overridden in `pgm_conf.json`. I don't know if the 'defined by the processor group' part helps the reader to better understand whatever the author wanted to focus on.

Also, 

> understood, but processor groups are not defined in pgm_conf.json, so I'd like to have those two words as far from the filename as possible.

Why this comment? I didn't reword the sentence so that one could imply that "processor groups are not defined in pgm_conf.json". The 'defined by the processor group' part was comma-separated

Reply:
Given that the sentence talks about overriding options, I think that the specification of what options are being overridden is actually useful.